Tools/TestWebKitAPI/Tests/WebKitCocoa/RunOpenPanel.mm

```diff
@@ -48,6 +48,7 @@ - (void)webView:(WKWebView *)webView runOpenPanelWithParameters:(WKOpenPanelPara
     EXPECT_FALSE(parameters.allowsDirectories);
     EXPECT_EQ(0ull, parameters._acceptedMIMETypes.count);
     EXPECT_EQ(0ull, parameters._acceptedFileExtensions.count);
+    [[NSFileManager defaultManager] createFileAtPath:[NSTemporaryDirectory() stringByAppendingPathComponent:expectedFileName] contents:nil attributes:nil];
```

Comment:
Maybe `removeItemAtPath` before the end of the test?

Reply:
Done!